.lagoon.yml

```diff
@@ -56,3 +56,12 @@ environments:
       schedule: "M/15 * * * *"
       command: drush cron
       service: cli
+    - name: import translations
+      # TODO Consider if translation checking frequency should be reduced.
+      # A high frequency is used to ensure quick turnaround time during
+      # development. Updating (or checking) translations could have side effects on
+      # caching etc. and once things have settled down a lower frequency may be
+      # more appropriate.
+      schedule: "M/30 * * * *"
```

Comment:
`"M/30 * * * *"` not sure what the `M` is here... not seen that in a cron looking definition before?